src/enrichment/crunchbase.py

```diff
@@ -0,0 +1,540 @@
+        investors[investor_cols].rename(columns={"id": "investor_id", "cb_url": "investor_url"}),
+        on="investor_id",
+        how="left",
+    )
+
+
+def _process_keywords(keywords: List[str], separator: str = ",") -> List[List[str]]:
+    """Process a list of keywords and keyword combinations
+
+    Args:
+        keywords (List[str]): list of keywords in the format ['keyword_1', 'keyword_2, keyword_3']
+
+    Returns:
+        List[List[str]]: list of lists of keywords in the format ['keyword_1', ['keyword_2', 'keyword_3']]
+    """
+    return [[word.strip() for word in line.split(separator)] for line in keywords]
+
+
+def get_keywords(
+    keyword_type: str,
+) -> Dict:
+    """
+    Fetch keywords from a Google Sheet
+
+    Keywords types correspond to missions and are ASF, AFS, AHL, and X.
+    """
+    # Process keywords
+    keywords_df = google.access_google_sheet(
+        os.environ["SHEET_ID_KEYWORDS"],
+        keyword_type,
+    )
+    return (
+        keywords_df.assign(keywords_list=lambda df: _process_keywords(df["Keywords"].to_list()))
+        .groupby("Subcategory")
+        .agg(keywords=("keywords_list", list))
+        .to_dict()["keywords"]
+    )
+
+
+def _enrich_funding_smart_money(
+    funding_rounds_enriched: pd.DataFrame,
+    investments: pd.DataFrame,
+    investors: pd.DataFrame,
+) -> pd.DataFrame:
+    """Enrich the funding rounds with smart money information"""
+    # Fetch the manually curated smart money table
+    smart_money_manual_df = google.access_google_sheet(os.environ["SHEET_ID_INVESTORS"], "investors")
+    return funding_rounds_enriched.pipe(
+        get_funding_round_investors, investments=investments, investors=investors
+    ).assign(
+        # Check if investment was made by one of the smart money types
+        smart_money_auto=lambda df: df.investor_types.str.contains("|".join(SMART_MONEY_TYPES)),
+        # Check if investment was made by one of the manually curated smart money investors
+        smart_money_manual=lambda df: df.investor_url.isin(smart_money_manual_df["Crunchbase URL"]),
+        # Combine the two smart money columns
+        smart_money=lambda df: df.smart_money_auto | df.smart_money_manual,
+    )
+
+
+def enrich_funding_rounds(
+    funding_rounds: pd.DataFrame,
+    investments: pd.DataFrame,
+    investors: pd.DataFrame,
+    funding_round_ids: Iterator[str] = None,
+    cutoff_year: int = 2000,
+) -> pd.DataFrame:
+    """Enrich the funding rounds with additional information"""
+    # If no IDs are specified, assume all funding rounds are needed
+    if funding_round_ids is None:
+        funding_round_ids = funding_rounds.id.unique()
+
+    return (
+        funding_rounds.query("id in @funding_round_ids")
+        # More informative column names
+        .rename(
+            columns={
+                "id": "funding_round_id",
+                "name": "funding_round_name",
+            }
+        )
```

Comment:
FYI @sqr00t, here I'm renaming the `id` in funding_rounds to `funding_round_id` 
Just something to remember when looking at diffs for funding_rounds table